core/src/common/context-options.ts

```diff
@@ -1,3 +1,7 @@
 export enum ContextOptions {
-  DISABLE_CONSTRUCTS_DEPLOYMENT_TRACKING = '@aws-analytics-reference-architecture/disableConstructsDeploymentTracking'
-}
\ No newline at end of file
+  DISABLE_CONSTRUCTS_DEPLOYMENT_TRACKING = '@aws-analytics-reference-architecture/disableConstructsDeploymentTracking',
+  EMR_EKS_TRACKING_ID = 'uksb-1sljh1bin',
+  DATA_ENG_PLATFORM_ID = 'uksb-1sljh1big',
```

Comment:
can we use the same code for EMR on EKS and Data_eng_platform? So we can keep one of them for future use

Reply:
I would keep DATA_ENG_PLATFORM_ID and EMR_EKS_TRACKING_ID. The EMR_EKS_TRACKING_ID can be used without the DATA_ENG and we would want to keep distinct usage statistics for both.

Reply:
For AraBucket we will include it once cdk-cleaning is merged with main. 

For DataGenerator it is going to be deprecated soon in the next releases, and we will be only using the replayer, should we still track it?

Reply:
I wanted to be frugal on codes but no prob if we can have more. You are right about DataGenerator